### PR TITLE
fixed headers for mobile and default layouts

### PR DESCRIPTION
**Pull request by nirshabi10**

### CODE REVIEW

File: src/views/BuyToLet/components/btlResult.jsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ class Result extends Component {
     if(product) {
       const { clientsScreen } = this.props
       if(clientsScreen === 'mobile') {
-        let normalizedProduct = product.replace(/(\d).years/, '$1/Y')
```
Comment (vasser) [inline src/views/BuyToLet/components/btlResult.jsx]: you could use `/(\d).years/i` to support different variations

Comment (nirshabi10) [reply src/views/BuyToLet/components/btlResult.jsx]: you are correct but years / Years is the string we are injecting, with no connection the the product itself. So i rather leave it Upper case

File: src/views/BuyToLet/components/results.jsx [hunk 2]
```diff
@@ -30,37 +31,36 @@ class Results extends Component {
...
-			'pageTitle': 'results',
-			'someUsefulInformation': '123abd'
-			})
-		}
-		this.toggleVisibility()
-		let btn = document.getElementById('startCalculation')
+	async componentDidMount() {
+    const { clientsScreen } = this.state
+    let lastNewMortgageResults = orderBy(sortBy(this.props.clientDetails.lastNewMortgageResults, 'amount'), 'amount', 'desc')
+    const resultsHeadersModule = await (await import(`./layouts/${clientsScreen}HeadersRow.jsx`)).default
```
Comment (vasser) [inline src/views/BuyToLet/components/results.jsx]: do we need double await here?

Comment (vasser) [reply src/views/BuyToLet/components/results.jsx]: oh, yes, we need. my bad

**Pull request merged**